ContactDetailsApi.Tests/V2/Gateway/TenureDbGatewayTests.cs

```diff
@@ -48,29 +48,96 @@ protected virtual void Dispose(bool disposing)
                 _disposed = true;
             }
         }
-        private async Task InsertDataIntoDynamoDB(IEnumerable<TenureInformationDb> entities)
+        private void InsertDataIntoDynamoDB(IEnumerable<TenureInformationDb> entities)
```

Comment:
?

Reply:
https://github.com/LBHackney-IT/contact-details-api/pull/143/commits/63f8152b38967e1d594239ab2c427332fd1fb4a1

Reply:
nvm, didn't notice the `.GetAwaiter().GetResult();`